flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```

Comment:
how about not restricting to x86_64?

Reply:
My application relies on PyQt5 and my method of installation is via the pip wheels that are only available as x86_64. I attempted to build PyQt5 and QtWebEngine from source within the flatpak but it would through errors (PySide2 would also do the same).

Reply:
For QtWebEgnine the recommended way is to use the base app. For PySide2, there are plenty of app that build it,